pypit/armsgs.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, log=None, debug=None, verbosity=None, colors=True):
         # Initialize other variables
         # TODO: debug could just be develop=True or False
         self._debug = debug
```

Comment:
I had a feeling that we were removing this functionality? So, I think all references to `self._debug` can be removed.

Reply:
Actually it hasn't been fully removed yet.  Just the bit that was in `pypit.py` that skipped printing the Traceback if the mode wasn't set to `develop`.